chainer/functions/normalization/standardize.py

```diff
@@ -0,0 +1,93 @@
+import six
+
+from chainer import cuda
+from chainer import function_node
+import chainer.functions
+from chainer.utils import type_check
+
+
+class Standardize(function_node.FunctionNode):
+
+    """Standardization for `Weight standardization
+        <https://arxiv.org/abs/1903.10520>`_"""
+
+    def __init__(self, eps=1e-5):
+        self.eps = eps
+
+    def check_type_forward(self, in_types):
+        type_check.expect(in_types.size() == 1)
+        x_type, = in_types
+
+        type_check.expect(
+            x_type.dtype.kind == 'f',
+            x_type.ndim >= 2,
+        )
+
+    def _compute(self, xp, x):
+        # xp: numpy, cupy, or chainer.functions
+        axes = tuple(six.moves.range(1, len(x.shape)))
+        mu = xp.mean(x, axis=axes, keepdims=True)
+        x_mu = x - mu
+        squ_x_mu = xp.square(x_mu)
+        var = xp.mean(squ_x_mu, axis=axes, keepdims=True)
+        std = xp.sqrt(var + self.eps)
+        inv_std = 1. / std
+        x_hat = x_mu * inv_std
+        return x_mu, var, inv_std, x_hat
+
+    def forward(self, inputs):
+        self.retain_inputs((0,))
+        xp = cuda.get_array_module(*inputs)
```

Comment:
https://github.com/chainer/chainer/pull/6678/files#r270065458 is for this.
```suggestion
        xp = backend.get_array_module(*inputs)
```